src/Algebra/Properties/Semigroup/Reasoning.agda

```diff
@@ -0,0 +1,121 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- uv≈wuational reasoning for semigroups
```

Comment:
Typo?

Reply:
Yeah, looks like an unwitting `global-search-and-replace` snafu.